npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -1067,6 +1080,11 @@ def parsed_lockfile
           )
         end
 
+        sig { params(uri: T.nilable(String)).returns(String) }
+        def sanitize_uri(uri)
+          uri.to_s.gsub("%2f", "/").gsub("%2F", "/").gsub(/\s.+/, "")
+        end
```

Comment:
are we able to use a ruby's native package`URI` for sanitizing uri's rather that doing it ourselves?

Reply:
@robaiken , good suggestion , didn't look into the native package , fixed it